src/compiler/postprocess.rs

```diff
@@ -0,0 +1,156 @@
+use std::{collections::HashMap, io::{BufWriter, Write}, path::PathBuf, process::{Command, Stdio}, sync::{Arc, Mutex, MutexGuard}};
+
+use itertools::Itertools;
+use wildmatch::WildMatchPattern;
+
+use crate::Cli;
+
+/// How it will pass bash code to the postprocessor
+#[derive(Debug, Clone)]
+pub enum PostProcessorInput {
+    /// Passes the data to the postprocessor's stdin
+    Stdin
+}
+
+/// How it will get the processed code from the postprocessor
+#[derive(Debug, Clone)]
+pub enum PostProcessorOutput {
+    /// Reads postprocessor's stdout
+    Stdout
+}
```

Comment:
What's the point of these? What other variants can be added here?

Reply:
like pass as an argument `processor -arg "CODE"` or as a file or something like that

Reply:
No but why create stdout enum with just out as variant. Same for stdin

Reply:
@Ph0enixKM i removed those, please review again